### PR TITLE
Wraping up the Protips bounty

### PR DESCRIPTION
**Pull request by MaxIsJoe**
This PR finally addresses the last remaining things that are required for this to be done.

Firstly, the protip list ui is finally fixed and is functional
image

Secondly, while working on the last two remaining tips; I've tried improving the general visibility of text by adding an outline/shadow behind that text so that it would stand out more.
image

And lastly, Protips have moved to their own gameObject for better organization and to make it easier to copy/paste protip configs around for custom player prefabs in the future.
image

There's some cleanup as well in the background, as well as the introduction of `stationTimeInMinutes` which I hope we'd use to replace the broken display for our discord bots in the future.

### Changelog:

CL: [New] Added two new protips for new players who join the game.
CL: [New] Added three new protips for changing intents.
CL: [New] Added a new protip for how to stand up again after falling down.
CL: [Fix] Finally fixed the cursed protips list UI.

### CODE REVIEW

**Comment by corp-0** [comment]
Click and drag the game object to the project view to create a prefab from it. That way any changes there doesn't also modify the player prefab and you can avoid conflicts

**Comment by MaxIsJoe** [comment]
> Click and drag the game object to the project view to create a prefab from it. That way any changes there doesn't also modify the player prefab and you can avoid conflicts

Done

File: UnityProject/Assets/Scripts/Learning/ProtipObject.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ public class ProtipObject : MonoBehaviour
 	{
 		public ProtipSO TipSO;
 		[SerializeField, Tooltip("Does this tip object only trigger once then saved as so?")]
-		private bool triggerOnce = true;
+		private bool triggerOnce, removeAfterRemembering = true;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Learning/ProtipObject.cs]: define it on its own line, you psycho

File: UnityProject/Assets/Scripts/Managers/GameManager.cs [hunk 2]
```diff
@@ -229,28 +231,35 @@ private void LoadConfig()
...
 		UpdateManager.Remove(CallbackType.UPDATE, UpdateMe);
+		if ( CustomNetworkManager.IsServer ) UpdateManager.Remove(CallbackType.PERIODIC_UPDATE, UpdateMinutes);
 		EventManager.RemoveHandler( Event.Cleanup, ClientCleanupInbetweenScenes );
 		EventManager.RemoveHandler( Event.CleanupEnd, ClientCleanupEndRoundCleanups );
 		EventManager.RemoveHandler( Event.PostRoundStarted, ClientRoundStartCleanup );
-
 	}
 
-
+	private void UpdateMinutes()
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Managers/GameManager.cs:264]: why is it called ``UpdateMinutes`` if it is updating every second?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Managers/GameManager.cs:264]: oh, whoops. It should update every 60 seconds, not every second.

Comment (corp-0) [reply UnityProject/Assets/Scripts/Managers/GameManager.cs:264]: it is probably better to handle it in seconds. Doesn't your clock also display seconds?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Managers/GameManager.cs:264]: what clock?

File: UnityProject/Assets/Scripts/Learning/ProtipObject.cs (hunk 1 above)
Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Learning/ProtipObject.cs]: I'm Patrick baeman of coding

File: UnityProject/Assets/Scripts/Managers/GameManager.cs (hunk 2 above)
Comment (corp-0) [reply UnityProject/Assets/Scripts/Managers/GameManager.cs:264]: the wallmounted clocks. They are supposed to give the time but they just give you the value you set at round start. If you use StationTime instead it is going to actually work

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Managers/GameManager.cs:264]: Isn't this a bit unrelated to this PR?

Comment (corp-0) [reply UnityProject/Assets/Scripts/Managers/GameManager.cs:264]: well you added a new implementation to StationTime on **this pr**

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Managers/GameManager.cs:264]: Both are different from each other.

`StationTime` is the oldest value used to represent how long a round has been going.
`StationTimeInMinutes` is an integer value that keeps track of the round's time while avoiding a bug with `DateTime` that gives negative/reset numbers once 24 hours pass.

Clocks use neither, as they use their own role-play compliant variables that allow players to see time as if it's in the year 2500+ and are there to serve a completely different purpose and future use-cases.

Honestly, `StationTime` should be re-named to `RoundTime` but this confusion wouldn't be an issue even if the naming was bad if we had comments explaining the difference between all of them, but we'd rather have clean code that is hard to follow than ugly code that is detailed and doesn't waste anyone's time.

Comment (corp-0) [reply UnityProject/Assets/Scripts/Managers/GameManager.cs:264]: rename it to RoundTime.

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Managers/GameManager.cs:264]: done 👌

**Review by corp-0 — APPROVED** [review]
